Java/serialization/src/main/java/com/amazon/randomcutforest/serialize/json/v1/V1JsonToV2StateConverter.java

```diff
@@ -46,6 +48,15 @@ public RandomCutForestState convert(String json, Precision precision) throws IOE
         return convert(forest, precision);
     }
 
+    public RandomCutForestState convert(ArrayList<String> jsons, int numberOfTrees, Precision precision)
```

Comment:
why not make a more general version of merge which takes multiple RandomCutForestState objects and outputs a merged RandomCutForestState? that way only one merge for the current version is needed

Reply:
Do not follow ... that is what is happening. The older convert now calls this new convert with an array list of size 1.

Reply:
why not leave this converter do its job, V1SerializedRandomCutForest to RandomCutForestState and have a separate merge function that takes (List<RandomCutForestState) and outputs a merged RandomCutForestState so this merge function can work on any states wherever the state is directly from v2 or indirect converted from v1.

Reply:
:) This library has never recommended that RCF be split. This package exists because the RCF state was split. Split models are inefficient and those considerations may/may not have gone in when the models were split. This RCF 2.0 package builds half a bridge -- the remainder of the bridge has to be built by the calling code.

Reply:
the suggestion is for improving code modularity.

with a bunch of json, the caller can do covert to get a list of states, then pass to merge(list of states). as you can see, existing convert is not unchanged. only a new merge is added to only address the merge need, in comparison to a monolithic convertAndMerge function.

Reply:
that said, i won't block this but to raise it for discussion

Reply:
Ok. It is a good discussion to have. An iterated merge is a possibility -- but that means creating new state classes for the merge. That means new tests, new artifacts -- it can become a standard "wag the dog". This serialization package exists to get out of the business of json management and not to add more weight in this section.  This code is supposed to be run once -- I'm inclined to not overdevelop it.

Reply:
just a last word for this discussion, should the merge function just look like this without any new classes/types? And much of the code here can be moved there.
```
RandomCutForestState merge(List<RandomCutForestState>, int numTrees)
```

Reply:
unrelated minor. ArrayList should be List for general use.

Reply:
I see that the meta comment is not being acknowledged. This is not general use. Now the code is available, and some folks want to take responsibility for model decisions and create more state for running merges, and then fail because of distributed consistency issues, they are welcome to do so. Also, Precision should be a parameter, because as far as I understand, precision is being converted as well.

Reply:
I acknowledge this viewpoint. The difference is not a blocker. The discussion should give future readers enough information on this decision.

Reply:
the suggested method is
```
public RandomCutForestState convert(List<String> jsons, int numberOfTrees, Precision precision)
```